src/app/components/community/sidebar/sidebar.ts

```diff
@@ -68,22 +54,22 @@ export default class AppCommunitySidebar extends Vue {
 
 	readonly GJ_IS_CLIENT = GJ_IS_CLIENT;
 
-	@Watch('collaborators', { immediate: true })
+	@Watch('data.collaborators', { immediate: true })
```

Comment:
Shouldn't these two watches just be getters?

Reply:
I don't think so, it's because this component holds its own state for those two properties, but wants to overwrite with incoming prop data when it is provided with it.